scripts/eloot.lic

```diff
@@ -4160,7 +4170,7 @@ module ELoot
         next if $sell_ignore.include?(thing.id)
         next if ELoot.data.ready_list.find { |_k, v| v.id == thing.id }
         next if !ELoot.data.settings[:sell_exclude].empty? && thing.name =~ /#{ELoot.data.settings[:sell_exclude].join('|')}/
-        next if thing.name =~ /bound/
+        next if thing.name =~ /bound|shimmering \w+ orb/
```

Comment:
Wondering if these should be wrapped in appropriate anchors, so becomes:
```ruby
next if thing.name =~ /\bbound\b|^shimmering \w+ orb$/

Reply:
Agree, that makes sense.